locales/fr-FR/messages.po

```diff
@@ -1469,3 +1469,6 @@ msgstr "Zoom arrière"
 
 msgid "Test Part Properties"
 msgstr "Propriétés des pièces d'essai"
+
+msgid "New section"
```

Comment:
Did you add it manually?
If so, kindly run a translation-key-gathering script instead.

Reply:
Wait a bit, it still doesn't seem like you've ran the script here.
Please try running `sudo -u www-data php tao/scripts/taoTranslate.php -a compileall -e taoQtiTest` in the php container.

Reply:
@wazelin no sense do 
`php tao/scripts/taoTranslate.php -a compileall -e taoQtiTest`
if you didn't do 
`php tao/scripts/taoTranslate.php -a updateall -e taoQtiTest`
because you source message.po will not change without `update`
https://hub.taotesting.com/articles/documentation-for-core-components/internationalization-i18n

Reply:
You are totally right. Misinterpreted the action's meaning. Thank you!